### PR TITLE
feature(dbt projects): models and tests

### PR DESCRIPTION
**Pull request by LukaDevic**

### CODE REVIEW

**Review by beevital — APPROVED** [review]
LGTM

**Comment by beevital** [comment]
It would be nice to have mocked tests that will check data mapping between API response and go-structures.

File: dbt_project_create.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+type DbtProjectCreateService struct {
+	c             *Client
+	groupID       *string
+	dbtVersion    *string
+	gitRemoteUrl  *string
+	gitBranch     *string
+	defaultSchema *string
+	folderPath    *string
+	targetName    *string
+	threads       int
```
Comment (beevital) [inline dbt_project_create.go:18]: This should be pointer, otherwise we won't be able to omit the value.

**Review by beevital — CHANGES_REQUESTED** [review]

File: dbt_project_create.go (hunk 1 above)
Comment (LukaDevic) [reply dbt_project_create.go]: Done. :)

**Comment by LukaDevic** [comment]
> It would be nice to have mocked tests that will check data mapping between API response and go-structures.

Added. :)

**Review by beevital — APPROVED** [review]

**Pull request merged**